src/cmd/linuxkit/azure.go

```diff
@@ -0,0 +1,372 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"time"
+
+	"path/filepath"
```

Comment:
could we include the standard imports in line and not as a separate sections. None of the other code does this.